EncoreTickets.SDK.Tests/UnitTests/Checkout/CheckoutServiceApiTests.cs

```diff
@@ -0,0 +1,564 @@
+    {
+        #region Checkout
+
+        public static IEnumerable<TestCaseData> Checkout_IfBookingIsSet_CallsApiWithRightParameters = new[]
+        {
+            new TestCaseData(
+                new BookingParameters
+                {
+                    Reference = "8527089",
+                    ChannelId = "europa-test",
+                    Shopper = new Shopper
+                    {
+                        Email = "user@example.com",
+                        Title = "Miss",
+                        FirstName = "Shawn",
+                        LastName = "Butler",
+                        TelephoneNumber = "07882571812",
+                        ExternalId = "8263702"
+                    },
+                    BillingAddress = new Address
+                    {
+                        Line1 = "Barnard's Inn",
+                        Line2 = "86 Fetter Lane",
+                        PostalCode = "EC4A 1EN",
+                        City = "London",
+                        CountryCode = "GB",
+                        CountryName = "United Kingdom",
+                        StateOrProvince = "NY"
+                    },
+                    Origin = "https://example.com",
+                    RedirectUrl = "https://example.com",
+                    DeliveryMethod = DeliveryMethod.C,
+                    DeliveryCharge = 245,
+                    RecipientName = "Mr. Someone Else",
+                    GiftVoucherMessage = "Happy Birthday to you.",
+                    DeliveryAddress = new Address
+                    {
+                        Line1 = "Barnard's Inn",
+                        Line2 = "86 Fetter Lane",
+                        PostalCode = "EC4A 1EN",
+                        City = "London",
+                        CountryCode = "GB",
+                        CountryName = "United Kingdom",
+                        StateOrProvince = "NY"
+                    },
+                    HasFlexiTickets = true,
+                    PaymentType = PaymentType.Card
+                },
+                "{\"reference\":\"8527089\",\"channelId\":\"europa-test\",\"shopper\":{\"email\":\"user@example.com\",\"title\":\"Miss\",\"firstName\":\"Shawn\",\"lastName\":\"Butler\",\"telephoneNumber\":\"07882571812\",\"externalId\":\"8263702\"},\"billingAddress\":{\"line1\":\"Barnard's Inn\",\"line2\":\"86 Fetter Lane\",\"postalCode\":\"EC4A 1EN\",\"city\":\"London\",\"countryCode\":\"GB\",\"countryName\":\"United Kingdom\",\"stateOrProvince\":\"NY\"},\"origin\":\"https://example.com\",\"redirectUrl\":\"https://example.com\",\"deliveryMethod\":\"C\",\"deliveryCharge\":245,\"recipientName\":\"Mr. Someone Else\",\"giftVoucherMessage\":\"Happy Birthday to you.\",\"deliveryAddress\":{\"line1\":\"Barnard's Inn\",\"line2\":\"86 Fetter Lane\",\"postalCode\":\"EC4A 1EN\",\"city\":\"London\",\"countryCode\":\"GB\",\"countryName\":\"United Kingdom\",\"stateOrProvince\":\"NY\"},\"hasFlexiTickets\":true,\"paymentType\":\"Card\"}"
```

Comment:
Please make these readable

Reply:
It cannot be made more readable because it is the exact string used in a query, and this is verified in tests with a string used as the body of the query.